.github/workflows/test-redis-enterprise.yml

```diff
@@ -0,0 +1,66 @@
+name: RE Tests
+
+on:
+  push:
+    branches: [master, v9]
+  pull_request:
```

Comment:
What about on pull_request (period). Maybe we should do the same platform label thing we have elsewhere, or is that overkill?